proto/prysm/v1alpha1/wrapper/beacon_block.go

```diff
@@ -109,9 +109,9 @@ func (_ Phase0SignedBeaconBlock) PbAltairBlock() (*eth.SignedBeaconBlockAltair,
 	return nil, errors.New("unsupported altair block")
 }
 
-// PbMergeBlock is a stub.
-func (_ Phase0SignedBeaconBlock) PbMergeBlock() (*eth.SignedBeaconBlockMerge, error) {
-	return nil, errors.New("unsupported merge block")
+// PbBellatrixBlock is a stub.
+func (_ Phase0SignedBeaconBlock) PbBellatrixBlock() (*eth.SignedBeaconBlockBellatrix, error) {
+	return nil, errors.New("unsupported bellatrix block")
```

Comment:
```suggestion
	return nil, errors.New("unsupported Bellatrix block")
```
This is particularly optional cause altair just above should be Altair to me being a proper name, but that's not part of this PR and other members may disagree on form here.